### PR TITLE
verifying eishay@kifi.com so he can be the root of the delegation chain

### PR DESCRIPTION
**Pull request by 2is10**
@eishay

### CODE REVIEW

File: tools/etc/nginx/conf.d/shoebox.conf [hunk 1]
```diff
@@ -173,8 +173,9 @@ server {
     return 200 '<!doctype html>';
   }
 
-  location = /google6de4ad16957106ef.html {
-    return 200 'google-site-verification: google6de4ad16957106ef.html';
+  # site verification for eishay@kifi.com
+  location = /google25ae05cb8bf5b064.html {
+    return 200 'google-site-verification: google25ae05cb8bf5b064.html';
```
Comment (2is10) [inline tools/etc/nginx/conf.d/shoebox.conf:178]: @stkem Will this change automatically make it into our next shoebox AMI?

Comment (stkem) [inline tools/etc/nginx/conf.d/shoebox.conf:178]: No, the configuration files need to be updated manually on the AMI and all the running shoeboxen. Hopefully we'll have that issue addressed in the next 3-4 weeks, but for now that's what it is.

Comment (stkem) [inline tools/etc/nginx/conf.d/shoebox.conf:178]: I.e. yes, it will make it into the next AMI, but only when somebody decides to make a new one.

Comment (2is10) [inline tools/etc/nginx/conf.d/shoebox.conf:178]: Great, thanks. I’ve updated the six current shoeboxen. I hope that any new shoeboxen we spin up will come from a new AMI. :pray:

**Pull request merged**